apis/v1alpha1/tlsconfig_types.go

```diff
@@ -60,7 +80,7 @@ type TLSConfig struct {
 	// values.
 	//
 	// +optional
-	MinimumVersion *string `json:"minimumVersion" protobuf:"bytes,2,opt,name=minimumVersion"`
+	MinimumVersion *string `json:"minimumVersion" protobuf:"bytes,3,opt,name=minimumVersion"`
```

Comment:
I believe this was removed by https://github.com/kubernetes-sigs/service-apis/pull/257

Reply:
Yes, will fix in the next rebase.